bdsx/bds/implements.ts

```diff
@@ -459,7 +459,10 @@ Actor.prototype.getBlockTarget = procHacker.js('?getBlockTarget@Actor@@QEBA?AVBl
 Actor.prototype.isAttackableGamemode = procHacker.jsv("??_7Actor@@6B@", "?isAttackableGamemode@Actor@@UEBA_NXZ", bool_t, {this:Actor});
 Actor.prototype.isInvulnerableTo = procHacker.jsv("??_7Actor@@6B@", "?isInvulnerableTo@Actor@@UEBA_NAEBVActorDamageSource@@@Z", bool_t, {this:Actor}, ActorDamageSource);
 
-Mob.prototype.knockback = procHacker.jsv('??_7Mob@@6B@', '?knockback@Mob@@UEAAXPEAVActor@@HMMMMM@Z', void_t, {this:Mob}, Actor, int32_t, float32_t, float32_t, float32_t, float32_t, float32_t);
+const Mob$knockback = procHacker.jsv('??_7Mob@@6B@', '?knockback@Mob@@UEAAXPEAVActor@@HMMMMM@Z', void_t, {this:Mob}, Actor, int32_t, float32_t, float32_t, float32_t, float32_t, float32_t);
+Mob.prototype.knockback = function (source: Actor | null, damage: int32_t, xd: float32_t, zd: float32_t, power: float32_t, height: float32_t, heightCap: float32_t): void {
```

Comment:
what is the purpose of these changes?

Reply:
> > > > How should I write?
> > > > > I think it would be better if the first parameter was optional.
> > > 
> > > 
> > > Please check `Mob.prototype.konckback`. if you have a hard to understanding, that's ok. I will do that.
> > 
> > 
> > I added it.
> 
> nice.
> 
> but there is one more thing to fix. when it is a default parameter, it can be undefined and incompatible with `NativeClass`es, like `Actor`. so you have to do like `Spawner.prototype.spawnMob`

For the same as `Spawner.prototype.spawnMob`, did I do it wrong?

Reply:
it's okay in this case.
because in `knockback`, `source` can be `null`, but not `undefined`.
we can only consider `undefined`.

Reply:
I deleted it!